.do/db-cluster/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "db_cluster_hosts" {
```

Comment:
Same for this. We don't need the "fors" just the value.
Then we can get the values with `terraform output ...`
https://www.terraform.io/docs/commands/output.html